src/elm/aggregate.coffee

```diff
@@ -45,6 +49,14 @@ module.exports.Min = class Min extends AggregateExpression
     list = @source.execute(ctx)
     return null unless list?
     listWithoutNulls = removeNulls(list)
+
+    # Check for incompatible units and return null. We don't want to convert
+    # the units for Min/Max, so we throw away tue converted array if it succeeds
```

Comment:
Small typo: `tue` -> `the`

Reply:
Good catch! Fixed this one and the one below just now